R/htmlwidgets.R

```diff
@@ -269,17 +274,26 @@ shinyRenderWidget <- function(expr, outputFunction, env, quoted) {
         instance$elementId, "\"; Shiny doesn't use them"
       )
     }
-    x <- .subset2(instance, "x")
     deps <- .subset2(instance, "dependencies")
     deps <- lapply(
       htmltools::resolveDependencies(deps),
       shiny::createWebDependency
     )
-    evals = JSEvals(x)
-    list(x = x, evals = evals, deps = deps)
+    payload = c(createPayload(instance), list(deps = deps))
```

Comment:
@ramnathv Sorry I missed a minor problem here: there is no need to assign the list to `payload` here. It does not really hurt since `=` returns the RHS invisibly, so the returned value is still correct.